skimage/_shared/utils.py

```diff
@@ -35,21 +35,22 @@ class deprecate_arg:
 
     def __init__(self, arg_mapping, removed_version=None):
         self.arg_mapping = arg_mapping
-        self.warning_msg = "'%s' is a deprecated argument name for %s. "
+        self.warning_msg = ("'{old_arg}' is a deprecated argument name "
+                            "for `{func_name}`. ")
         if removed_version is not None:
-            self.warning_msg += "It will be removed in version %s" % (
-                removed_version)
-        self.warning_msg += "Use '%s' instead."
+            self.warning_msg += (f"It will be removed in version "
+                                 "{removed_version}. ")
+        self.warning_msg += "Use '{new_arg}' instead."
 
     def __call__(self, func):
         @functools.wraps(func)
         def fixed_func(*args, **kwargs):
             for old_arg, new_arg in self.arg_mapping.items():
                 if old_arg in kwargs:
                     #  warn that the function interface has changed:
-                    warnings.warn(self.warning_msg %
-                                  (old_arg, func.__name__, new_arg),
-                                  DeprecationWarning)
+                    warnings.warn(self.warning_msg.format(
+                        old_arg=old_arg, func_name=func.__name__,
+                        new_arg=new_arg), DeprecationWarning, stacklevel=2)
```

Comment:
DeprecationWarnings are really easy to silence. please use `FutureWarning`

Reply:
Done